ui/droppable.js

```diff
@@ -98,8 +99,6 @@ $.widget( "ui.droppable", {
 		var drop = $.ui.ddmanager.droppables[ this.options.scope ];
 
 		this._splice( drop );
-
-		this.element.removeClass( "ui-droppable ui-droppable-disabled" );
```

Comment:
Is "ui-droppable-disabled" never being added in the first place? Or do I have to search elsewhere?